idaes_examples/notebooks/docs/surrogates/sco2/omlt/properties.py

```diff
@@ -0,0 +1,316 @@
+
+        if state_vars_fixed is False:
+            if hold_state is True:
+                return flags
+            else:
+                blk.release_state(flags)
+
+    def release_state(blk, flags, outlvl=0):
+        '''
+        Method to relase state variables fixed during initialisation.
+
+        Keyword Arguments:
+            flags : dict containing information of which state variables
+                    were fixed during initialization, and should now be
+                    unfixed. This dict is returned by initialize if
+                    hold_state=True.
+            outlvl : sets output level of of logging
+        '''
+        if flags is None:
+            return
+
+        # Unfix state variables
+        for k in blk.keys():
+            if flags['Fcflag'][k] is False:
+                blk[k].flow_mol.unfix()
+            if flags['Pflag'][k] is False:
+                blk[k].pressure.unfix()
+            if flags['Tflag'][k] is False:
+                blk[k].temperature.unfix()
+
+        if outlvl > 0:
+            if outlvl > 0:
+                _log.info('{} State Released.'.format(blk.name))
+
+
+@declare_process_block_class("SCO2StateBlock",
+                             block_class=_StateBlock)
+class SCO2StateBlockData(StateBlockData):
+    """
+    An example property package for ideal gas properties with Gibbs energy
+    """
+
+    def build(self):
+        """
+        Callable method for Block construction
+        """
+        super(SCO2StateBlockData, self).build()
+        self._make_state_vars()
+
+    def _make_state_vars(self):        
+        self.flow_mol = Var(domain=NonNegativeReals,
+                            initialize=1.0,
+                            units=units.kmol/units.s,
+                            doc='Total molar flowrate [kmol/s]')
+        self.pressure = Var(domain=NonNegativeReals,
+                            initialize=8,
+                            bounds=(7.38, 40),
+                            units=units.MPa,
+                            doc='State pressure [MPa]')
+
+        self.temperature = Var(domain=NonNegativeReals,
+                               initialize=350,
+                               bounds=(304.2, 760+273.15),
+                               units=units.K,
+                               doc='State temperature [K]')
+
+        self.entr_mol = Var(domain=Reals,
+                            initialize=10,
+                            units=units.kJ/units.kmol/units.K,
+                            doc='Entropy [KJ/kmol/K]')
+        
+        self.enth_mol = Var(domain=Reals,
+                            initialize=1,
+                            units=units.kJ/units.kmol,
+                            doc='Enthalpy [KJ/ kmol]')
+ 
+        inputs=[self.pressure,self.temperature]
+        outputs=[self.enth_mol,self.entr_mol]
+        self.keras_surrogate = KerasSurrogate.load_from_folder("sco2_keras_surr")
```

Comment:
The same thing as for the Pysmo surrogate file applies - if you move the `sco2_keras_surr` directory inside the `omlt` subdirectory, the code should work without other changes.

Reply:
Thanks @lbianchi-lbl, the tests passed for pysmo and now the sco2_keras_surr is also being recognized but I am getting error with variables folder. I checked the omlt example of auto-thermal reformer and I have two similar files in my folder as well. Can you please help me with this ?

Reply:
Ah. Sorry. Its again the variable folder was not added in the fork. I will retry and let you know. I have another question regarding the positioning of this files. The other examples have them in the parent directory which failed for me, do you know a potential reason for this ? 

<img width="735" alt="image" src="https://github.com/IDAES/examples/assets/73403218/e2e3c7f7-938d-4b5d-a50a-f6342696590a">

Reply:
I think I understand the general issue you're referring to, but I'm not sure I can find specific examples. From what I can see in for `idaes_examples/notebooks/docs/surrogates`, in both the `pysmo` and `omlt` subfolder, the same notebook generates the model and loads it from a folder, as opposed to using a pre-existing file: 
 
![image](https://github.com/IDAES/examples/assets/48035537/42a5737a-01ab-4604-b440-a81a196b6b97)
![image](https://github.com/IDAES/examples/assets/48035537/314bf3e1-057b-4028-b8b9-15104b9c23ff)

Reply:
Oh ok. So that might be the case.